mmdeploy/codebase/mmdet3d/deploy/mmdet3d.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmengine.registry import Registry
+
+from mmdeploy.codebase.base import CODEBASE, MMCodebase
+from mmdeploy.utils import Codebase
+
+MMDET3D_TASK = Registry('mmdet3d_tasks')
+
+
+@CODEBASE.register_module(Codebase.MMDET3D.value)
+class MMDetection3d(MMCodebase):
+    """MMDetection3d codebase class."""
+
+    task_registry = MMDET3D_TASK
+
+    @classmethod
+    def register_deploy_modules(mmdet3d):
```

Comment:
This is a class method and the first argument should be `cls` by default.

Reply:
ok